### DISCUSSION TITLE
Linux file system or IoT file support

### DISCUSSION THREAD

**Discussion by bos0407** [thinkst/canarytokens#244] [Feature Requests]
The canarytoken is very useful for the security operators, and I found some tokens are just supported Windows file system (e.g., PDF, MS Word, Window Directory, EXE file).
Do any plans to support the Linux file system or IoT?

**Comment by jayjb** [comment]
Hi @bos0407,

Thats a very interesting thought. Are there any particular file types you think are valuable for Linux or IoT? We can then do some research into making it happen.

**Comment by bos0407** [comment]
Normally, most of MIS is setting the Linux-based devices to be a file sharing service not only on PC-Based servers but also on IoT devices. 
If any hacker interests the file in the storage of Linux-based server, they will encrypt or exposures those files and threaten the victim.
So any sensitive files including PDF, XLS, DOC, the document has touched, then trigger the alert system and warning to the system manager.
If it is possible, the research will be interested.

**Comment by kevin-dall** [comment]
IMHO, I would think the most valuable things within a linux system would be .conf file you could put in /etc. This is where the shadow file is for Linux password hashes. Would not see MS office files in a juicy location like /etc in Linux or .pdf for that matter. With years of pentesting under my belt, I would see those files as a red flag in juicy locations in Linux

**Comment by shadow-absorber** [comment]
using linux as a daily driver here so placing some pdf:s that work on linux would be nice... together with checking for possible commands being run like whoami or checking for people reading files in /etc like for example make a dummy /etc/shadow.bak
